modules/oe_content_call_proposals/config/install/field.field.node.oe_call_proposals.oe_call_proposals_journal.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_call_proposals_journal
+    - node.type.oe_call_proposals
+  module:
+    - link
+id: node.oe_call_proposals.oe_call_proposals_journal
+field_name: oe_call_proposals_journal
+entity_type: node
+bundle: oe_call_proposals
+label: 'Publication in the official journal'
+description: "The link title is limited to 128 characters maximum.\r\nTitle: Official Journal reference.\r\nURL: link to the Official Journal."
+required: false
+translatable: false
```

Comment:
This field is translatable according to requirements.

Reply:
@imanoleguskiza we have agreed to set `translatable: true` in `field.storage.*` configurations only.

Reply:
This is changed, @imanoleguskiza should I revert the setting back to `false`?

Reply:
Indeed this is my bad, only old content types are marked as translatable, which is something we will need to fix in another ticket.